### DISCUSSION TITLE
Transformation d'images

### DISCUSSION THREAD

**Discussion by victorjourne** [ia-foule/MobileCount#39] [Ideas]
Discussion à propos des **transformations** d'images dans les loaders par dataset

**Comment by victorjourne** [comment]
Meilleure transfo pour SSHB (MAE Golden **56** au lieu de 62): 
```
own_transforms.RandomDownOverSampling(4),
own_transforms.RandomCrop((576,768)),
own_transforms.RandomHorizontallyFlip()
```

**Comment by fabricejourdan** [comment]
Meilleure transformation aux votes pour WE (MAE Golden 71 (*))
own_transforms.RandomCrop(cfg_data.TRAIN_SIZE),
own_transforms.RandomHorizontallyFlip()
(*) Les résultats de la baseline n'ont jamais été retrouvé (size WE et pas SHHB), sans relu, et randomcrop avec error)
